packages/react-components/package.json

```diff
@@ -70,7 +70,7 @@
     "@babel/cli": "^7.24.8",
     "@babel/core": "^7.26.10",
     "@babel/preset-env": "7.23.9",
-    "@playwright/experimental-ct-react": "~1.50.1",
+    "@playwright/experimental-ct-react": "~1.51.1",
     "playwright-core": "~1.50.1",
```

Comment:
Please update the other playwright dependencies too (including the ones used in composites UI tests)